### PR TITLE
[onert/test] Update nnpackage test script

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit updates nnpackage test scripts.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: https://github.com/Samsung/ONE/pull/10496
Related issue: https://github.com/Samsung/ONE/issues/9877

### CODE REVIEW

File: tests/scripts/benchmark_nnpkg.sh [hunk 1]
```diff
@@ -128,7 +120,7 @@ for i in "${model_lists[@]}"; do
 
   echo "" >> ${outfile}
 
-  TFLITE_CMD="LD_LIBRARY_PATH=./Product/out/lib THREAD=3 ${tflite_run} -r 10 -m 1 -p 1"
+  TFLITE_CMD="THREAD=3 ${tflite_run} -r 10 -m 1 -p 1"
```
Comment (glistening) [inline tests/scripts/benchmark_nnpkg.sh:123]: Is `LD_LIBRARY_PATH` not necessary any longer?

Comment (hseok-oh) [reply tests/scripts/benchmark_nnpkg.sh:123]: Right.

https://github.com/Samsung/ONE/blob/497fcd27fbd9cd935296e913664a09f32cbfbb62/infra/nnfw/CMakeLists.txt#L9-L10

`CMAKE_INSTALL_RPATH` helps to find `*.so` libraries.

Comment (glistening) [reply tests/scripts/benchmark_nnpkg.sh:123]: I confimed `LD_LIBRARY_PATH` was not necessary before this PR though it is removed by this PR.

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**